demo/scenario/script

```diff
@@ -9,36 +9,35 @@
 
 scenario
 
-# to start we either have to
-# (a) manually create auction (iff so, uncomment below) ..
-# wait "create auction manuall and enter auction id" auctionId
-# or (b) auto-create auction (iff so, uncomment below)
+# create auction
 submit Auctioneer1 createAuction
 
+wait "Auction Created. Maybe show auctioneers auction view?  (Hit any key to continue)"
+
 # clock-rounds ...
 submit Auctioneer1 startNextRound
 submit A-Telecom submitClockBid
 submit B-Net submitClockBid
 submit C-Mobile submitClockBid
 submit Auctioneer1 endRound
 
-wait "End Round 1  (Hit any key to continue)"
+say "End Round 1"
 
 submit Auctioneer1 startNextRound
 submit A-Telecom submitClockBid
 submit B-Net submitClockBid
 submit C-Mobile submitClockBid
 submit Auctioneer1 endRound
 
-wait "End Round 2  (Hit any key to continue)"
+yell "End Round 2"
```

Comment:
I guess a `say` would also work here ....

Reply:
I put a yell as round number seemed relevant to know for the subsequent manual step. But had no strong feelings. Just thought we probably do not want a wait here but still somewhat highlight the new round ...

Reply:
ok